packages/spacecat-shared-data-access/test/it/v2/index.test.js

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/* eslint-env mocha */
+/* eslint-disable no-console */
+
+import AWSXRay from 'aws-xray-sdk';
+import { expect, use } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { spawn } from 'dynamo-db-local';
+import { v4 as uuid, validate as uuidValidate } from 'uuid';
+
+import { isIsoDate } from '@adobe/spacecat-shared-utils';
+import SCHEMA from '../../../docs/schema.json' with { type: 'json' };
+import { sleep } from '../../unit/util.js';
+import { dbClient } from '../db.js';
+import { createTable, deleteTable } from '../tableOperations.js';
+import { createDataAccess } from '../../../src/service/index.js';
+
+use(chaiAsPromised);
+
+const DATA_TABLE_NAME = 'spacecat-services-data';
+
+const setupDb = async (client, table) => {
+  await deleteTable(client, table);
+
+  const schema = SCHEMA.DataModel.find((model) => model.TableName === table);
+  await createTable(client, schema);
+};
+
+const generateSampleData = async (dataAccess, siteId) => {
+  const { Opportunity, Suggestion } = dataAccess;
+  const sampleData = { opportunities: [], suggestions: [] };
+
+  for (let i = 0; i < 10; i += 1) {
+    const type = i % 2 === 0 ? 'broken-backlinks' : 'broken-internal-links';
+    const status = i % 2 === 0 ? 'NEW' : 'IN_PROGRESS';
+    const data = type === 'broken-backlinks'
+      ? { brokenLinks: [`https://another-example-${i}.com`] }
```

Comment:
It's a confusing example, since the data for broken backlinks won't look like this, so I'd just put some foo-bar properties for the moment.